src/org/usfirst/frc/team2335/robot/commands/Lift.java

```diff
@@ -0,0 +1,41 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class Lift extends Command
+{
+    public Lift() 
+    {
+        requires(Robot.climb);
+    }
+
+    
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute() 
+    {
+    	//Starts the climb process
+    	Robot.climb.startClimb();
+    }
+    
+    protected boolean isFinished() 
+    {
+        return false;
```

Comment:
Does this mean the climb motor will run until match end / robot power-down? If so, we might over-stress the motor when there's no more climbing to do. Alternatively, might I be mis-reading this?